readthedocs/builds/forms.py

```diff
@@ -173,7 +173,7 @@ def clean_match_arg(self):
         except Exception:
             raise forms.ValidationError(
                 _('Invalid Python regular expression.'),
-            )
+            ) from None
```

Comment:
If we are going to use `from None` on these, shouldn't we use the context? Like:

```python
        except Exception as e:
            raise forms.ValidationError(
                _('Invalid Python regular expression.'),
            ) from e
```

Reply:
I'm just going to remove this rule